src/App.js

```diff
@@ -1,24 +1,17 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Switch, Route } from 'react-router-dom';
+import Home from './pages/home/Home';
+import ContactUs from './pages/contact/ContactUs';
+import nav from './pages/home/nav'
```

Comment:
this nav component name MUST start with a capital letter, besides it is imported but  never used